packages/react-toast/src/hooks.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import {Toast} from '@shopify/polaris';
```

Comment:
I'm not sure I like this depending on polaris. It would be cool if we just had the provider take a `renderToast` function.